inc/debug.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#if defined(CH_CFG_TIME_QUANTUM)
+// We are compiling for ChibiOS
+#include "osal.h"
+#define debugAssert(condition, remark) osalDbgAssert(condition, remark)
+#define debugCheck(condition) osalDbgCheck(condition)
+#else
+// We are not compiling for ChibiOS
+#include <assert.h>
```

Comment:
#include \<cassert\>
https://stackoverflow.com/questions/5197136/why-is-there-a-c-prefix-in-the-c-version-of-c-libs